.eslintrc

```diff
@@ -24,21 +26,39 @@
       }
     ],
     "import/no-internal-modules": [
-      "warn", {
+      "warn",
+      {
         "allow": [
           "hyperview/**",
           "react-native/Libraries/StyleSheet/StyleSheetTypes"
         ]
       }
-    ]
+    ],
+    "max-len": [
+      "warn",
+      {
+        "code": 100
+      }
+    ],
+    "import/no-cycle": "off",
+    "lines-between-class-members": "off",
+    "no-else-return": "off",
+    "no-restricted-globals": "off",
+    "prefer-destructuring": "off",
+    "react/destructuring-assignment": "off",
+    "react/jsx-sort-props": "off",
+    "react/no-access-state-in-setstate": "off",
+    "sort-keys": "off"
```

Comment:
I think some of them should be added to `eslint-plugin-instawork` instead of hyperview repo individually. Could you also consult @flochtililoch for these changes? Some of them were intentionally added in `eslint-plugin-instawork`.

Reply:
See my other comments - these are intended to be temporary.